EIPS/eip-2657.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 2657
+title: Ephemeral Testnet Yolo
+author: James Hancock (@madeoftin)
+discussion-to: https://gitter.im/ethereum/AllCoreDevs
+status: Draft
+type: Meta
+created: 2020-04-19
+---
+
+**Disclaimer: This is for testing basic infrastructure. It will be nuked. It is not for deploying dapps, nor does it define what will go into mainnet. For information on network upgrades, please follow the relevant meta EIPs and ongoing discussion on Ethereum/pm.**
+
+## Abstract
+
+The specification for Ephemeral Testnet Yolo. Clients who wish to sync need to implement the following features into their client. It is for testing basic infrastructure and will be nuked.
+
+## Specification 
+
+Name: Yolo
+ID: `YOLO-v1`
+
+  - [x] EIP 2537 Commit Hash - [5edff4ae6ff62c7e0bbfad624fc3d0ba7dc84392](https://github.com/ethereum/EIPs/commit/5edff4ae6ff62c7e0bbfad624fc3d0ba7dc84392)
+  - [x] EIP 2315 Commit Hash - [e8accf22cdc5562d6982c560080c6cd6b7f94867](https://github.com/ethereum/EIPs/commit/e8accf22cdc5562d6982c560080c6cd6b7f94867)
+
+*[ ] Proposed - [x] Consensus to include.*
+## Timeline
+
+ - Deployed: June 3rd 2020
+ 
+## Client Consensus -> Implementation 
+
+YOLO-v1
+| **Client**   | Signal | Spec | Merged | Syncing |
+|--------------|--------|------|--------|---------|
+| Besu         | x      | x    |        |         |
+| EthereumJS   | x      |      |        |         |
+| Geth         | x      | x    | x      | x       |
+| Nethermind   | x      | x    |        |         |
+| OpenEthereum | x      | x    |        |         |
+| Trinity      |        |      |        |         |
+
+**Signal** -
+Client intends to participate. *(You are on the bus)*
+
+**Spec** -
+Client is statisfied with the proposed specification. *(You agree with the direction)*
```

Comment:
"Statisfied" is causing the codespell error here

Reply:
Good find Thank you!

Reply:
updated